README.md

```diff
@@ -1,49 +1,75 @@
 # GovWifi Authentication API
 
-Authentication API for GovWifi
-
-## Overview
+This API is used by the frontend when a connection request is made to GovWifi.
+Its function is to check these requests against user details and the result determines how FreeRadius reacts to it e.g. allow / dissallow the connection.
+
+The private GovWifi [build repository][build-repo] contains instructions on how to build GovWifi end-to-end - the sites, services and infrastructure.
+
+<!-- vscode-markdown-toc -->
+* 1. [Overview](#Overview)
+	* 1.1. [Sinatra routes](#Sinatraroutes)
+	* 1.2. [Dependencies](#Dependencies)
+* 2. [Developing](#Developing)
+	* 2.1. [Running the tests](#Runningthetests)
+	* 2.2. [Using the linter](#Usingthelinter)
+	* 2.3. [Serving the app locally](#Servingtheapplocally)
+	* 2.4. [Deploying changes](#Deployingchanges)
+* 3. [Gotchas](#Gotchas)
+	* 3.1. [Extra API parameters](#ExtraAPIparameters)
+* 4. [Licence](#Licence)
+
+<!-- vscode-markdown-toc-config
```

Comment:
This looks like it's for vscode specifically, and shouldn't actually be committed. If I paste this into a general markdown editor, it shows!